### ISSUE TITLE
Require maas nodes to be tagged with `cluster-api` or something in order to be selected

### ISSUE THREAD

**Issue by foolusion** [samsung-cnct/cma-ssh#90]
the maas client should only use nodes that the user allows it too. We should require all nodes to be tagged with some magic string to identify that they are allowed to be used. I mentioned this in #87

**Comment by oneilcin** [comment]
I agree, but also wonder if various users will want to tag them differently.  If we are coming up with standard types for a customer - I think we need to talk to them about the types of machines they have and what tags make sense.

**Comment by oneilcin** [comment]
Oh you meant like a constant tag, separate from the instanceType type tag.  Makes sense
